iesg_statements/2007-03-20-0.md

```diff
@@ -64,7 +64,7 @@ Date: Mar 2007
 
    This section provides some guidelines for the use of the AD Sponsoring method.  Such guidelines are useful when authors contact the AD and suggest that their document be sponsored.  The rules are also useful in controlling the load on the IESG, and to ensure fairness.  AD Sponsored documents are the only way to publish Standards Track documents outside WGs.  IETF documents may also have a higher priority at the RFC Editor processing queue than independent submissions.
 
-   When considering the choice between a sponsored document and an RFC Editor submission, the RFC 3932 rules play a role RFC3932.  If it is likely that the document would generate a 3 (harmful to IETF work), 4 (violates IETF procedures) or 5 (extension requires IETF review) response based on RFC 3932 it is not appropriate for an independent submission.  Sometimes such documents are suitable candidates for being sponsored, however.  It would be useful to add, say, IANA rules or IPv6 considerations to an old specification that did not have them and for which no WG can be found.  Such additions to standards track RFCs need to be on the standards track themselves, preventing the use of independent submissions.
+   When considering the choice between a sponsored document and an RFC Editor submission, the RFC 3932 rules play a role [RFC3932].  If it is likely that the document would generate a 3 (harmful to IETF work), 4 (violates IETF procedures) or 5 (extension requires IETF review) response based on RFC 3932 it is not appropriate for an independent submission.  Sometimes such documents are suitable candidates for being sponsored, however.  It would be useful to add, say, IANA rules or IPv6 considerations to an old specification that did not have them and for which no WG can be found.  Such additions to standards track RFCs need to be on the standards track themselves, preventing the use of independent submissions.
```

Comment:
we can drop the redundant [RFC3932] here.

Reply:
Dropped.